.stylelintrc.json

```diff
@@ -6,8 +6,7 @@
 	],
 	"extends": [
 		"stylelint-config-recommended",
-		"stylelint-config-sass-guidelines",
-		"stylelint-config-prettier"
```

Comment:
What issue was this causing?

Reply:
The rules these modified were deprecated in the newest version of stylelint. It made ~50 'unkown rule' errors per file unless they were removed.